README.md

```diff
@@ -19,7 +21,9 @@ Use this template to scaffold a new website
 4. Change minimum stability to dev: `"minimum-stability": "dev",`
 5. `composer require remihin/filament-prefab`
 6. Install all modules:
-- `php artisan prefab:filament --module=base --force` (be patient with the shell script, force is required to overwrite the user model)
+- `php artisan prefab:filament --module=base --force`
+  - be patient with the shell script, force is required to overwrite the user model
+  - This installs curator, make sure the published migration will be executed before other migrations (a simple rename will suffice)
```

Comment:
Kan je dit nog automatiseren? Want is nu niet duidelijk wat ik precies moet doen in welke stap